src/main/java/controller/LoginController.java

```diff
@@ -20,8 +20,8 @@ void doPost(HttpRequest request, HttpResponse response) throws Exception {
         response.getHeaders().setCookie(cookie);
 
         // session 변경
-        HttpSession session = new HttpSession();
-        HttpSessionStorage httpSessionStorage = new HttpSessionStorage();
+        HttpSession session = request.createHttpSession();
+        HttpSessionStorage httpSessionStorage = request.getSessionStorage();
```

Comment:
```suggestion
        HttpSession session = request.getSession();
```
LoginController에서 직접 세션을 만들지 않고 관리하지 않아도 자동으로 세션을 만들어서 반환하게 만들어보면 어떨까요? 🤔

Reply:
세션 생성 책임을 컨트롤러가 아니라 HttpRequest 클래스에서 담당하게 하라는 말씀이신거죠 ??
리팩토링하겠습니다~